app/src/main/res/layout/dialog_info_map_mode.xml

```diff
@@ -0,0 +1,152 @@
+            android:layout_weight="1"
+            android:paddingEnd="@dimen/full_padding"
+            android:src="@drawable/ic_online"
+            tools:ignore="ContentDescription,RtlSymmetry" />
+
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="8"
+            android:orientation="vertical"
+            android:paddingTop="@dimen/quarter_padding">
+
+            <TextView
+                style="@style/TextAppearance.AppCompat.Title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingBottom="@dimen/quarter_padding"
+                android:text="@string/online"
+                android:textSize="16sp" />
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/online_description" />
+        </LinearLayout>
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/hybrid_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/online_layout"
+        android:orientation="horizontal"
+        android:paddingHorizontal="@dimen/full_padding"
+        android:paddingBottom="@dimen/half_padding">
+
+        <ImageView
+            android:layout_width="0dp"
+            android:layout_height="20dp"
+            android:layout_weight="1"
+            android:paddingEnd="@dimen/full_padding"
+            android:src="@drawable/ic_hybrid"
+            tools:ignore="ContentDescription,RtlSymmetry" />
+
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="8"
+            android:orientation="vertical">
+
+            <TextView
+                style="@style/TextAppearance.AppCompat.Title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingBottom="@dimen/quarter_padding"
+                android:text="@string/hybrid"
+                android:textSize="16sp" />
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/hybrid_description" />
+        </LinearLayout>
+
+    </LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/offline_layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/hybrid_layout"
+        android:orientation="horizontal"
+        android:paddingHorizontal="@dimen/full_padding"
+        android:paddingBottom="@dimen/half_padding">
+
+        <ImageView
+            android:layout_width="0dp"
+            android:layout_height="20dp"
```

Comment:
There are still many values that do not use dimens.xml, check all dp and sp sizes